Directory.Build.props

```diff
@@ -327,6 +326,15 @@
     <EnableMultiFileILCTests Condition="'$(ConfigurationGroup)|$(ArchGroup)' == 'Release|x64' Or '$(ConfigurationGroup)|$(ArchGroup)' == 'Debug|x86' Or '$(ConfigurationGroup)|$(ArchGroup)' == 'Release|arm'">true</EnableMultiFileILCTests> -->
   </PropertyGroup>
 
+  <!-- Specify the default constants for each target group -->
+  <PropertyGroup>
+    <DefineConstants Condition="'$(TargetsUap)' == 'true'">$(DefineConstants);uap</DefineConstants>
+    <DefineConstants Condition="'$(TargetsNetCoreApp)' == 'true'">$(DefineConstants);netcoreapp</DefineConstants>
```

Comment:
The default meaning of `netcoreapp` in corefx is current version of netcoreapp. This is changing the meaning to any netcoreapp version. This confusing, I think.

Reply:
Ah, will change it. I was not sure since there was at least one definition that matched the above.

Reply:
Ok, changed it back. Now src/shims/manual/Directory.Build.props references the new definition. I am not sure if it has any effect or not, but I would be really happy if the definitions could be consistent across the whole code base.